lib/galaxy/datatypes/molecules.py

```diff
@@ -1043,132 +1028,15 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
-        if not dataset.dataset.purged:
-            dataset.peek = get_file_peek(dataset.get_file_name())
-            dataset.info = self.get_dataset_info(dataset.metadata)
-            structure_string = "structure" if dataset.metadata.number_of_molecules == 1 else "structures"
-            dataset.blurb = f"CIF file containing {dataset.metadata.number_of_molecules} {structure_string}"
-
-        else:
-            dataset.peek = "file does not exist"
-            dataset.blurb = "file purged from disk"
-
-    def get_dataset_info(self, metadata):
-        if self.meta_error:
-            info = "Error finding metadata. The file may be formatted incorrectly."
-        elif ase_io:
-            # enhanced info
-            info_list = []
-            if metadata.number_of_molecules == 1:
-                info_list.append(f"Chemical formula:\n{metadata.chemical_formula[0]}")
-                if metadata.is_periodic[0]:
-                    info_list.append("Periodic:\nYes")
-                    info_list.append(
-                        f"Lattice parameters in axis-angle format:\n{', '.join([str(round(x, 2)) for x in metadata.lattice_parameters[0]])}"
-                    )
-                else:
-                    info_list.append("Periodic:\nNo")
-                info_list.append(f"Atoms in file:\n{metadata.number_of_atoms[0]}")
-            elif metadata.number_of_molecules > 1:
-                info_list.append("File contains multiple structures; full metadata will not be displayed.")
-                formulae = "\n".join(metadata.chemical_formula)
-                info_list.append(f"Chemical formula for each structure in this file:\n{formulae}")
-            info = "\n--\n".join(info_list)
-        else:
-            info = """
-Metadata is limited as the Atomic Simulation Environment (ASE) is not installed.
-You can still use this dataset in tools and workflows.
-For full metadata, ask your admin to install the 'ase' Python package."""
-
-        return info
-
 
 @build_sniff_from_prefix
-class XYZ(GenericMolFile):
+class XYZ(AtomicStructFile):
     """
     XYZ format.
     """
 
     file_ext = "xyz"
-    meta_error = False
-    MetadataElement(
-        name="atom_data",
-        default=[],
-        desc="Atom symbols and positions",
-        readonly=True,
-        visible=False,
-    )
-    MetadataElement(
-        name="number_of_atoms",
-        desc="Number of atoms",
-        readonly=True,
-        visible=True,
-    )
-    MetadataElement(
-        name="chemical_formula",
-        desc="Chemical formula",
-        readonly=True,
-        visible=True,
-    )
-    MetadataElement(
-        name="is_periodic",
-        desc="Periodic boundary conditions",
-        readonly=True,
-        visible=True,
-    )
-    MetadataElement(
-        name="lattice_parameters",
-        desc="Lattice parameters",
-        readonly=True,
-        visible=True,
-    )
+    ase_format = "xyz"
```

Comment:
```suggestion
    # ASE recommend always parsing as extended xyz
    ase_format = "extxyz"
```

As the comment above notes, ASE recommends using `extxyz` regardless. Also, since `ExtendedXYZ` inherits this, currently we will never load the lattice parameters correctly as we don't try to. This manifests as `extxyz` files appearing without `pbc` or lattice vectors in your history.